assignments/lambda-classes.js

```diff
@@ -1 +1,177 @@
 // CODE here for your Lambda Classes
+// CODE here for your Lambda Classes
+class Person {
+    constructor(personAttr) {
+      this.name = personAttr.name;
+      this.age = personAttr.age;
+      this.location = personAttr.location;
+      this.gender = personAttr.gender;
+    }
+  
+    speak() {
+      console.log(`Hello, my name is ${this.name}, I am from ${this.location}`);
+    }
+  }
+
+class Instructor extends Person {
+    constructor(instructorAttr) {
+        super(instructorAttr);
+        this.specialty = instructorAttr.specialty;
+        this.favLanguage = instructorAttr.favLanguage;
+        his.catchPhrase = instructorAttr.catchPhrase;
```

Comment:
When this is your only flaw, you're in good shape

```suggestion
        this.catchPhrase = instructorAttr.catchPhrase;
```